contributing/content-style.md

```diff
@@ -174,6 +175,20 @@ Use                                                                         | Av
 It's a new key management service (KMS). You should use this KMS carefully. | It's a new KMS. You should use this KMS carefully.
 You have many service options, including Elasticsearch and Kafka.           | You have many service options, e.g. Elasticsearch, Kafka, etc.
 
+## Use the long form of commands
+
+The Platform.sh CLI has aliases for certain commands to make typing shorter.
+Some of these are semantically valid, but some might be confusing.
+
+Always use the long form of the command to show the entire context, including the namespace.
+Exception: don't both with the `auth` namespace as `login` and `logout` are clear enough without additional context.
```

Comment:
The exception is about the namespace, so I'll try to reword there. Also to avoid the typo (`both` for `bother`). The suggestion seems to be `don't use`, which I think isn't what's intended. :slightly_smiling_face: